src/main/java/competition/subsystems/drive/swerve/SwerveSteeringSubsystem.java

```diff
@@ -80,6 +80,7 @@ public SwerveSteeringSubsystem(SwerveInstance swerveInstance, XCANSparkMaxFactor
         if (electricalContract.isDriveReady()) {
             this.motorController = sparkMaxFactory.create(electricalContract.getSteeringNeo(swerveInstance), this.getPrefix(), "SteeringNeo");
             setMotorControllerPositionPidParameters();
+            this.motorController.setSmartCurrentLimit(40);
```

Comment:
Suggest adding a comment to the effect of how this number was chosen like in the pr description